### DISCUSSION TITLE
Under the hood of github-readme-stats

### DISCUSSION THREAD

**Discussion by anuraghazra** [anuraghazra/github-readme-stats#1318] [Show and tell]
Checkout this amazing post by @Bogdan-Lyashenko where he wrote about how github-readme-stats works in-depth.

[Under the hood of github-readme-stats project](https://codecrumbs.io/library/github-readme-stats)

[![](https://res.cloudinary.com/anuraghazra/image/upload/v1632328461/underthehoodofgrs_tjvm5y.png)](https://codecrumbs.io/library/github-readme-stats)

Thank you for writing this! ❤️ 
And huge shoutout to the [codecrumbs](https://github.com/Bogdan-Lyashenko/codecrumbs) project! It's soo cool 🤩

**Comment by Bogdan-Lyashenko** [comment]
Thanks! Well, besides github-readme-stats being very popular to use (obviously 😺 ) it is also very interesting for developers, because it is kind of "next generation" for app building; you know, with Vercel, lambda functions and integration into Github and everything. It caught my curiosity and I think might be helpful for other developers as well, to lower the threshold into contributing to the codebase or just with understanding how this was built!
